lib/livebook_web/live/hooks/user_hook.ex

```diff
@@ -45,6 +43,8 @@ defmodule LivebookWeb.UserHook do
         attrs -> attrs
       end
 
+    user = User.new(attrs["id"])
```

Comment:
This is already handled by `update_user` below, no?

Reply:
No because we use this `user` when it returns the `{:error, changeset}` tuple

Reply:
Ah, good call :)